website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,121 @@
+- After dbt completes building the models, the MetricFlow Server processes the exports, compiles the necessary datasets, and executes data operations.
+- You can review the exports execution details in the jobs logs. Since saved queries are integrated into the dbt DAG, all outputs related to exports are available in the `dbt build` logs.
+- Your data is now available in the data platform for querying.
+
+   <Lightbox src="/img/docs/dbt-cloud/semantic-layer/saved_queries_run.jpg" width="90%" title="TeThe st"/>
+
+
+## Caching
+
+Caching reduces load times and costs by pre-computing and storing frequently queried datasets. This feature is critical for companies with large datasets or requiring fast query compute times.
+
+There are two types of caching:
+- Result cache &mdash; Cache all queries in the data platform and configured at the project or environment-level. This cache is used to store the results of a query save costs, and speed up subsequent queries.
+- Declarative cache &mdash; Saved queries specified by the user and defaults to all saved queries being cached or not. This cache enables more control over what is cached.
+
+To configure caching, you can add the `config` property to your `saved_queries` configuration:
+
+```yaml
+saved_queries:
+  - name: my_query
+    description:
+    query_params:
+      ...
+    exports:
+      ...
+    config:
+      cache:
+        enabled: true|false
+```
+
+### Parameters
+
+| GraphQL	| dbt Cloud CLI |	ADBC| Description	|
+| --- | --- | --- | --- |
+| `create-query` | `dbt sl query` |	`{{ semantic_layer.query() }}` | Allows you to build the SQL for a query that does not hit the cache, utilizes a saved-query to specify parameters or specifies whether the cache should be utilized or overwritten.
+| `drop-cache` |	`dbt sl cache drop` |	`{{ semantic_layer.drop_cache() }}` |	Drops datasets from the cache using parameters which target specific object |
+| `inspect-cache` |	`dbt sl cache inspect` |	`{{ semantic_layer.inspect_cache() }}` |	Inspect and provides a list of objects that are in the cache. | 
+
+## Job commands?
+
+Additional parameters and commands are introduced in MetricFlow CLI and Cloud Interfaces to support Saved Queries and Exports functionalities.
+
+| GraphQL	| dbt Cloud CLI |	ADBC	| Description	| 	
+| --- | --- | --- | --- |
+| `create-query` |	`dbt sl query` |	`{{ semantic_layer.query() }}` |	Allows you to build the SQL for a query that does not hit the cache, utilizes a saved-query to specify parameters or specifies whether the cache should be utilized or overwritten	|
+| `export`	| `dbt sl export`	| None, Not possible	| Builds an export using the GraphQL API. Use the export [parameters](#parameters). specified above in  |
+
+
+## FAQs
```

Comment:
maybe we can add some common faqs we've seen? like how does this relate to other cloud features/products (e.g. cloud cli, explorer)

Reply:
Maybe an FAQ question could be like, "How do Exports help me use the dbt Semantic Layer in tools that don't support it, such as PowerBI?"